packages/react-slider/src/components/Slider/Spec.md

```diff
@@ -136,42 +119,21 @@ https://hackmd.io/VUpPADJ7Ry-ZXTrtffD7Sg
 
   ```jsx
   <slots.root {...slotProps.root}>
-    {state.marks && <slots.marksContainer {...slotProps.marksContainer} />}
-    <slots.sliderWrapper {...slotProps.sliderWrapper}>
```

Comment:
Will the wrapper probably be needed again later if we add marks back? Historically we've tried to avoid semi-dramatic DOM changes like that.

Reply:
I believe we can add marks back in with a single slot addition (one of the advantages to using grid layout). So I'm confident leaving all of the wrappers out.

Reply:
Also, after conversation with design, it looks like marks are intentionally omitted from this control.